protocols/imap/src/main/java/org/apache/james/imap/processor/AppendProcessor.java

```diff
@@ -26,6 +26,8 @@
 
 import javax.mail.Flags;
 
+import org.apache.commons.io.IOUtils;
+import static org.apache.commons.io.output.NullOutputStream.NULL_OUTPUT_STREAM;
```

Comment:
We tend to position static imports above normal  imports

Reply:
Done